crates/bevy_pbr/src/render/light.rs

```diff
@@ -699,8 +715,7 @@ pub fn prepare_lights(
     point_light_shadow_map: Res<PointLightShadowMap>,
     directional_light_shadow_map: Res<DirectionalLightShadowMap>,
     mut shadow_render_phases: ResMut<ViewBinnedRenderPhases<Shadow>>,
-    mut max_directional_lights_warning_emitted: Local<bool>,
```

Comment:
Started to fail compilation after merging in main, I think it was due to many arguments tripping the compiler out:

```
error[E0599]: the method `in_set` exists for fn item `fn(Commands<'a, 'b>, ..., ..., ..., ..., ..., ..., ..., ..., ..., ..., ..., ..., ..., ..., ..., ...) {prepare_lights}`, but its trait bounds were not satisfied
   --> crates/bevy_pbr/src/lib.rs:401:26
```

Easiest way around it I could find was to merge these two locals into a container struct.